ui/app/components/ui/EditButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "~/components/ui/button";
+import { Plus, Trash2 } from "lucide-react";
+import {
+  Tooltip,
+  TooltipContent,
+  TooltipTrigger,
+} from "~/components/ui/tooltip";
+
+interface DeleteButtonProps {
+  label?: string;
+  onDelete?: () => void;
+}
+
+export function DeleteButton({ label, onDelete }: DeleteButtonProps) {
```

Comment:
I would say just split these into separate files so that we can easily find these